package/scie-pants.toml

```diff
@@ -0,0 +1,229 @@
+[lift]
+name = "scie-pants"
+description = "Isolates your Pants from the elements."
+load_dotenv = true
+
+[lift.app_info]
+repo = "https://github.com/pantsbuild/scie-pants"
+
+[lift.ptex]
+id = "ptex"
+version = "0.7.0"
+lazy_argv1 = "{scie.env.PANTS_BOOTSTRAP_URLS={scie.lift}}"
+
+[lift.scie-jump]
+version = "0.11.0"
+
+[[lift.interpreters]]
+id = "cpython38"
+provider = "PythonBuildStandalone"
+release = "20230507"
+lazy = true
+version = "3.8.16"
+
+[[lift.interpreters]]
```

Comment:
N.B.: Adding 3.11 support just requires adding a similar table with "3.11.3" as the version, an id of "cpython311" and an edit to `tools/src/scie_pants/configure_pants.py` that selects "cpython311" for the appropriate Pants version cutoff.